matRad/gui/widgets/matRad_PlanWidget.m

```diff
@@ -30,6 +30,7 @@
     
     properties (Constant)
         Modalities = {'photons','protons','carbon', 'helium'};
+        AvailableProjections = {  'physicalDose'; 'RBExD'; 'effect'; 'BED'; }
```

Comment:
I actually now realize that I don't like that some properties of classes have a capital first letter, this violates our camelCase coding style in matRad.
You can change it here, but probably we need to do a clean revision (maybe there's a tool for that).